safe-global/web-core
# fix: strikethrough other transactions on hover

## What it solves

Resolves #699

## How this PR fixes it

When hovering over the execution icon or button on conflicting transactions, the `id` of that transaction is saved. Other transactions in the group are strikedthrough in reference to this (and pending transactions of the same group).

A new "Transaction will be replaced" status has been added as well.

## How to test it

Create a group of conflicting transactions. Observe when hovering over the execute icon or button that the other transactions in the group are strikethrough. When one of them is execute and is pending, the other remain strikedthrough.

## Screenshots

### Hovering

![image](https://user-images.githubusercontent.com/20442784/192753468-8ef281dd-cb49-4728-8c48-bb814340fb85.png)

![image](https://user-images.githubusercontent.com/20442784/192753441-1611c680-0b3a-4e4b-9288-1a19687ae578.png)

![image](https://user-images.githubusercontent.com/20442784/192753427-ac8e128d-d3de-4a22-868e-715d02ac905f.png)

![image](https://user-images.githubusercontent.com/20442784/192753409-c5f7c9c5-9a34-4256-a079-5d6cf8806b85.png)

### Executing

![image](https://user-images.githubusercontent.com/20442784/192753371-4a0955bf-4424-468e-9266-c72676eab242.png)
